configs/workflows-mt.yaml

```diff
@@ -0,0 +1,377 @@
+        data_object_type: Crispr Terms
+        description: Crispr Terms for {id}
+        name: Crispr Terms
+      - output: stats_tsv
+        data_object_type: Annotation Statistics
+        description: Annotation Stats for {id}
+        name: Annotation statistics report
+      - output: map_file
+        data_object_type: Contig Mapping File
+        description: Contig mappings file for {id}
+        name: Contig mappings between contigs and scaffolds
+      - output: imgap_version
+        data_object_type: Annotation Info File
+        description: Annotation info for {id}
+        name: File containing annotation info
+      - output: renamed_fasta
+        data_object_type: Assembly Contigs
+        description: Assembly contigs (remapped) for {id}
+        name: File containing contigs with annotation headers
+
+
+  - Name: Expression Analysis Antisense
+    Type: nmdc:MetatranscriptomeExpressionAnalysis
+    Enabled: True
+    Git_repo: https://github.com/microbiomedata/metaT_ReadCounts
+    Version: v0.0.1
+    WDL: readcount.wdl
+    Collection: metatranscriptome_expression_analysis_set
+    Predecessors:
+    - Metagenome Annotation
+    Input_prefix: nmdc_expression
+    Inputs:
+      gff_file: do:Functional Annotation GFF
+      map: do:Contig Mapping File
+      bam: do:Assembly Coverage BAM
+      rna_type: "aRNA"
+      proj: "{activity_id}"
+    Activity:
+      name: "Metatranscriptome Expression Analysis for {id}"
+      type: nmdc:MetatranscriptomeExpressionAnalysis
+    Outputs:
+    - output: count_table
+      data_object_type: Metatranscriptome Expression 
+      description: Expression counts for {id}
+      name: Metatranscriptome expression table
+    - output: count_ig
+      data_object_type: Metatranscriptome Expression Intergenic
+      description: Metatranscriptome expression intergenic regions for {id}
+      name: Metatranscriptome expression intergenic table
+    - output: info_file
+      data_object_type: Metatranscriptome Expression Info File
+      description: Expression info for {id}
+      name: Metatranscriptome Expression Info File
+
+  - Name: Expression Analysis Sense
+    Type: nmdc:MetatranscriptomeExpressionAnalysis
+    Enabled: True
+    Git_repo: https://github.com/microbiomedata/metaT_ReadCounts
+    Version: v0.0.1
+    WDL: readcount.wdl
+    Collection: metatranscriptome_expression_analysis_set
+    Predecessors:
+    - Metagenome Annotation
+    Input_prefix: nmdc_expression
+    Inputs:
+      gff_file: do:Functional Annotation GFF
+      map: do:Contig Mapping File
+      bam: do:Assembly Coverage BAM
+      proj: "{activity_id}"
+    Activity:
+      name: "Metatranscriptome Expression Analysis for {id}"
+      type: nmdc:MetatranscriptomeExpressionAnalysis
+    Outputs:
+    - output: count_table
+      data_object_type: Metatranscriptome Expression
+      description: Expression counts for {id}
+      name: Metatranscriptome expression table
+    - output: count_ig
```

Comment:
@scanon  what is the syntax for an output being optional?

Reply:
```suggestion
    - output: count_ig
    required: false
```

Reply:
It actually should be "optional: true"